runtime/autoload/zip.vim

```diff
@@ -83,29 +83,22 @@ fun! zip#Browse(zipfile)
    return
   endif
 
-  let repkeep= &report
-  set report=10
+  let dict = s:SetSaneOpts()
 
   " sanity checks
-  if !exists("*fnameescape")
-   if &verbose > 1
-    echoerr "the zip plugin is not available (your vim doesn't support fnameescape())"
-   endif
-   return
-  endif
   if !executable(g:zip_unzipcmd)
    redraw!
-   echohl Error | echo "***error*** (zip#Browse) unzip not available on your system"
-   let &report= repkeep
+   echohl Error | echomsg "***error*** (zip#Browse) unzip not available on your system"
```

Comment:
How about defining functions to show error or warning messages?

Reply:
good idea.